tests/ha/test_multi_server_pod_failure.py

```diff
@@ -266,15 +272,17 @@ def test_reads_kserver_pods_fail(self):
             LOGGER.debug("Services status on %s : %s", pod_name, resp)
             if not resp[0]:
                 counter += 1
-            pod_list.remove(pod_name)
+            server_data_pod_list.remove(pod_name)
```

Comment:
You have to add data_pod_list + server_pod_list.
Here I guess server_data_pod_list contain only data pods.